apps/web/src/features/enrollments/components/detail-sections/enrollment-basic-info.tsx

```
@@ -0,0 +1,149 @@
+							>
+								<X className="h-4 w-4" />
+							</Button>
+						</div>
+					)}
+				</CardTitle>
+			</CardHeader>
+			<CardContent className="space-y-4">
+				<div>
+					<label className="font-medium text-muted-foreground text-sm">
+						Status
+					</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)